lib/ui/myplot/PlotList.dart

```diff
@@ -158,10 +158,10 @@ class PlotList extends StatelessWidget {
     BuildContext context,
     ViewModelProvider<RecommendationsListViewModel> provider,
   }) {
-    Navigator.of(context).push(
-      MaterialPageRoute(
-        builder: (context) => RecommendationsList(provider: provider),
-      ),
+    showDialog<bool>(
```

Comment:
can this be added to some kind of helper, something like NavigationScope.showModal(context, widget) ?

Reply:
I'm refactoring the NavigationScope (because the android back button was not working).
I will open the other PR and I will merge the other one first and then I will rebase and add the function in this PR.

Reply:
👍

Reply:
Done